diff --git a/Source/HBIOS/md.asm b/Source/HBIOS/md.asm
index 2b9c9c0..8bc3bee 100644
--- a/Source/HBIOS/md.asm
+++ b/Source/HBIOS/md.asm
@@ -5,7 +5,11 @@
 ;
 ; MD DEVICE CONFIGURATION
 ;
+#IFDEF MDRAMDISABLE
+MD_DEVCNT	.EQU	1		; NUMBER OF MD DEVICES SUPPORTED
+#ELSE
 MD_DEVCNT	.EQU	2		; NUMBER OF MD DEVICES SUPPORTED
+#ENDIF
 MD_CFGSIZ	.EQU	6		; SIZE OF CFG TBL ENTRIES
 ;
 MD_DEV		.EQU	0		; OFFSET OF DEVICE NUMBER (BYTE)
@@ -15,10 +19,12 @@ MD_LBA		.EQU	2		; OFFSET OF LBA (DWORD)
 ; DEVICE CONFIG TABLE (RAM DEVICE FIRST TO MAKE IT ALWAYS FIRST DRIVE)
 ;
 MD_CFGTBL:
-	; DEVICE 1 (RAM)
+#IFNDEF MDRAMDISABLE
+	; DEVICE 1 (RAM)		; ROM IF
 	.DB	1			; DRIVER DEVICE NUMBER
 	.DB	0			; DEVICE STATUS
 	.DW	0,0			; CURRENT LBA
+#ENDIF
 	; DEVICE 0 (ROM)
 	.DB	0			; DEVICE NUMBER
 	.DB	0			; DEVICE STATUS
@@ -34,23 +40,32 @@ MD_CFGTBL:
 ;
 MD_INIT:
 	CALL	NEWLINE			; FORMATTING
+#IFDEF MDRAMDISABLE
+	PRTS("MD: UNITS=1 $")
+#ELSE
 	PRTS("MD: UNITS=2 $")
+#ENDIF
 	PRTS("ROMDISK=$")
 	LD	HL,ROMSIZE - 128
 	CALL	PRTDEC
+
+#IFNDEF MDRAMDISABLE
 	PRTS("KB RAMDISK=$")
 	LD	HL,RAMSIZE - 128
 	CALL	PRTDEC
 	PRTS("KB$")
+#ENDIF
 ;
 ; SETUP THE DIO TABLE ENTRIES
 ;
 	LD	BC,MD_FNTBL
 	LD	DE,MD_CFGTBL
+#IFNDEF MDRAMDISABLE
 	PUSH	BC
 	CALL	DIO_ADDENT
 	POP	BC
 	LD	DE,MD_CFGTBL + MD_CFGSIZ
+#ENDIF
 	CALL	DIO_ADDENT
 ;
 	XOR	A			; INIT SUCCEEDED
diff --git a/Source/HBIOS/romldr.asm b/Source/HBIOS/romldr.asm
index 8449108..ed9336c 100644
--- a/Source/HBIOS/romldr.asm
+++ b/Source/HBIOS/romldr.asm
@@ -549,7 +549,11 @@ romload1:
 	; Record boot information
 	pop	af			; recover source bank
 	ld	l,a			; L := source bank
-	ld	de,$0100		; boot volume/slice
+#IFDEF MDRAMDISABLE
+	ld	de,$0000		; boot volume/slice (A)
+#ELSE
+	ld	de,$0100		; boot volume/slice (B)
+#ENDIF
 	ld	b,BF_SYSSET		; HBIOS func: system set
 	ld	c,BF_SYSSET_BOOTINFO	; BBIOS subfunc: boot info
 	rst	08			; do it
