diff --git a/Source/HBIOS/Build.sh b/Source/HBIOS/Build.sh
index 91d4e22..7f2be2a 100755
--- a/Source/HBIOS/Build.sh
+++ b/Source/HBIOS/Build.sh
@@ -65,9 +65,8 @@ romfmt=wbw_rom${romsize}
 outdir=../../Binary
 
 echo "creating empty rom disk of size $romsize in $blankfile"
-#LANG=en_US.US-ASCII tr '\000' '\345' </dev/zero | dd of=$blankfile bs=1024 count=`expr $romsize - 128` 2>/dev/null
-#LC_CTYPE=en_US.US-ASCII tr '\000' '\345' </dev/zero | dd of=$blankfile bs=1024 count=`expr $romsize - 128` 2>/dev/null
-LC_ALL=en_US.US-ASCII tr '\000' '\345' </dev/zero | dd of=$blankfile bs=1024 count=`expr $romsize - 128`
+dd of=$blankfile bs=1024 count=`expr $romsize - 128` < <(LC_CTYPE=en_US.US-ASCII tr '\000' '\345' </dev/zero)
+
 hexdump $blankfile
 
 cat <<- EOF > build.inc
