diff --git a/Source/HBIOS/romldr.asm b/Source/HBIOS/romldr.asm
index aa8455b..9a8272f 100644
--- a/Source/HBIOS/romldr.asm
+++ b/Source/HBIOS/romldr.asm
@@ -550,13 +550,6 @@ reboot:
 ;=======================================================================
 ;
 romload:
-;
-	; Notify user
-	ld	hl,str_load
-	call	pstr
-	ld	l,(ix+ra_name)
-	ld	h,(ix+ra_name+1)
-	call	pstr
 ;
 #if (DSKYENABLE)
 	ld	hl,msg_load		; point to load message
